real_time/arduino_src/lib_avr/rbserialmessages/rbsm_headers.txt

```diff
@@ -0,0 +1,11 @@
+RBSM_MID_ENC_TICKS_LAST, 0
```

Comment:
lets just get that written and tested (before) we pull this in.  A pull request has been made for the high level code